app/src/routes/project/LayersPanel/index.tsx

```diff
@@ -42,7 +93,7 @@ const LayersPanel = observer(() => {
                         <LayersTab />
                     </TabsContent>
                     <TabsContent value={TabValue.COMPONENTS}>
-                        <div className="w-full pt-96 text-center opacity-70">Coming soon</div>{' '}
```

Comment:
Maybe leave this as a comment until the feature is ready. Usually would remove but will be toggling this for publishing.

Reply:
https://github.com/onlook-dev/onlook/pull/285/commits/37ac194f14a789cf1e6f051897e0ee4723a75d8f